doc/formatting-code.md

```diff
@@ -0,0 +1,55 @@
+# Formatting Code
+
+As of this writing, there are NPM scripts in `package.json` that automatically
+format your code for you using Prettier and ESLint. These scripts use
+prettier-eslint, which is a package that "formats your code via `prettier`, and
+then passes the result of that to `eslint --fix`." This way we get the benefits
+of Prettier's superior formatting capabilities and the configuration
+capabilities of ESLint.
+
+The following are the NPM scripts:
+
+```json
+    "format-check-ts": "prettier-eslint \"src/**/*.ts\"",
+    "format-write-ts": "prettier-eslint \"src/**/*.ts\" --write",
+    "format-check-vue": "prettier-eslint \"src/**/*.vue\"",
+    "format-write-vue": "prettier-eslint \"src/**/*.vue\" --write",
+    "format-check": "npm run format-check-ts && npm run format-check-vue",
+    "format-write": "npm run format-write-ts && npm run format-write-vue"
+```
+
```

Comment:
I would mildly recommend quoting all of this code in the docs here because (a) you've described the behavior of each script in detail below, and (b) it just becomes another thing to sync manually when the relevant code changes. If we adopt a doc framework (see #71) that allows automated code snippeting (that's automatically kept up to date), I wouldn't object to such code quotes in the docs when useful. If you feel strongly that the docs should have the code here because it will be helpful to people trying to use the tools, then please in the relevant code section (in package.json, I guess) comment that any changes there need to be propagated to this doc. Otherwise they will absolutely fall out of sync eventually. Thanks.

Reply:
The snippet was removed in https://github.com/numberscope/frontscope/pull/90/commits/45a1d13411b7e0681d276678db36656bd017c48f.